game/exceptions.py

```diff
@@ -34,3 +34,18 @@ class NotOwnerException(APIException):
 class GameSessionFullException(APIException):
     status_code = 400
     default_detail = {'Game Session': ['Game session is full.']}
+
+
+class BuildingNameException(APIException):
+    status_code = 400
+    default_detail = {'Building': ['Building name is invalid.']}
+
+
+class ResourcesException(APIException):
+    status_code = 400
+    default_detail = {'Resources': ['You do not have enough resources.']}
```

Comment:
I would change the name of the Exception to be more meaningful: `NotEnoughResourcesException` or `InsufficientResourcesException`